h2o-algos/src/main/java/hex/tree/DTree.java

```diff
@@ -1652,20 +1662,21 @@ static Split findBestSplitPointUplift(DHistogram hs, int col, double min_rows) {
     double prY1CT1 = nCT1Y1hi/nCT1;
     double prY1CT0 = nCT0Y1hi/nCT0;
     double bestUpliftGain = upliftMetric.node(prY1CT1 , prY1CT0);
+    double upliftGainBefore = bestUpliftGain;
     // if there are any NAs, then try to split them from the non-NAs
     if (wNA>=min_rows) {
       double prCT1All = (nCT1 + numTreatNA + 1)/(nCT0 + numContrNA + nCT1 + numTreatNA + 2);
       double prCT0All = 1-prCT1All;
-      double prY1CT1All = (resphiTreat[0] + respTreatNA) / (nCT1 + numTreatNA);
-      double prY1CT0All = (resphiContr[0] + respContrNA) / (nCT0 + numContrNA);
+      double prY1CT1All = (nCT1Y1hi + respTreatNA) / (nCT1 + numTreatNA);
```

Comment:
I know it probably seems obvious for you what the `nCT1Y1hi` means but would you please at least comment near the variable declaration what it is supposed to mean, i.e., how to decode the name to some meaningful description (e.g., imagine I have a variable `nCY1` which can be decoded as  "number of Control data points with Y ==1").

To be clear, I am perfectly ok with using short variable names if it is easy to decode them.

Reply:
Sure, I will do it. Thanks!